### PR TITLE
Implement logging out of other logged-in clients

### PR DESCRIPTION
**Pull request by alayek**
Raising pull request as a simple implementation of #1725 after discussion with @marceloschmidt .

### CODE REVIEW

File: packages/rocketchat-ui-account/account/accountProfile.html [hunk 1]
```diff
@@ -59,6 +59,9 @@
 				<div class="submit">
 					<button class="button"><i class="icon-send"></i><span>{{_ "Save_changes"}}</span></button>
 				</div>
+				<div class="logoutOthers">
+					<button class="button">Logout Others</button>
```
Comment (rodrigok) [inline packages/rocketchat-ui-account/account/accountProfile.html]: Can you translate this string?

Just replace `Logout Others` by `{{Logout_Others}}` and add this key and text in `i18n/en.i18n.json`

Comment (sampaiodiego) [inline packages/rocketchat-ui-account/account/accountProfile.html]: Better than this, just use the Blaze helper, like this:
`{{_ "Logout_Others"}}`

Comment (rodrigok) [inline packages/rocketchat-ui-account/account/accountProfile.html]: Yes, sorry, I would like to wrote what @sampaiodiego wrote

Comment (alayek) [inline packages/rocketchat-ui-account/account/accountProfile.html]: Ok, I can do that and raise a PR. However, now my `development` branch is out of sync with the upstream repo. 

How do I pull in changes and update both my forked repo and cloned repo; without losing my changes? I have a feeling I might need to `git rebase` and / or `git stash`; though I am not exactly sure about the steps.

Also, what about Strings in other languages? Do I also have to provide those? Or would we leave that as another issue, for future work, to be merged from another PR?

File: packages/rocketchat-ui-account/account/accountProfile.coffee [hunk 2]
```diff
@@ -106,3 +106,9 @@ Template.accountProfile.onRendered ->
 Template.accountProfile.events
 	'click .submit button': (e, t) ->
 		t.save()
+	'click .logoutOthers button': (event, templateInstance) ->
+		Meteor.logoutOtherClients (error) -> 
+			if error
+				toastr.error error.reason
+			else
+				toastr.success t('Logged out of other clients successfully')
```
Comment (marceloschmidt) [inline packages/rocketchat-ui-account/account/accountProfile.coffee]: This string must also be added to `i18n/en.18n.json`. Please change it to `t('Logged_out_of_other_clients_successfully')` and add it to the translations file.

File: packages/rocketchat-ui-account/account/accountProfile.html (hunk 1 above)
Comment (rodrigok) [inline packages/rocketchat-ui-account/account/accountProfile.html]: @alayek you don't need to create a new PR, just change the files in your branch `logout-other-clients` commit and push then this PR will be updated.

Just for english, to add the new field to be translated to other languages at the LingoHup website.

**Comment by alayek** [comment]
I feel like I have done something horribly wrong with this pull request, by merging the `development` branch into my feature branch... I will delete the forked repo and create a new pull request and all. Hope that's ok.

**Comment by engelgabriel** [comment]
closed in favor of https://github.com/RocketChat/Rocket.Chat/pull/1896

**Pull request closed without merging**